### PR TITLE
Improve Cloud Starter - New Relic config instructions

### PR DESCRIPTION
**Pull request by meker12**
Per user feedback (PR #6708):

Improve instructions for configuring Starter projects for New Relic reporting:
-  Add prerequisites for credentials and role access required to add
the license key to an environment
-  Link to alternative instructions for getting the New Relic license
key from New Relic account, and adding the license key from the Magento
Admin
-  Added separate H3 headings for Configure Starter for New Relic and
Remove license key from Starter to make content easier to scan

## Purpose of this pull request
Per user feedback (PR #6708)

Improve instructions for configuring Starter projects for New Relic reporting
-  Add prerequisites for credentials and role access required to add
the license key to an environment
-  Link to alternative instructions for getting the New Relic license
key from New Relic account, and adding the license key from the Magento
Admin
-  Added separate H3 headings for Configure Starter for New Relic and
Remove license key from Starter to make content easier to scan

## Affected DevDocs pages

https://devdocs.magento.com/cloud/project/new-relic.html

whatsnew
Improved the instructions to configure Cloud Starter projects to use New Relic to add alternative instructions for users who have access to New Relic and the Magento Admin for a site, but do not have access to configure environment variables through the Magento Cloud project. See [Configure New Relic for Starter environments](https://devdocs.magento.com/cloud/project/new-relic.html#configure-new-relic-for-starter-environments).

### CODE REVIEW

File: src/cloud/project/new-relic.md [hunk 1]
```diff
@@ -69,40 +69,52 @@ runtime:
...
+-  For Starter accounts, you have a New Relic license key that supports up to three environments. You must add the key to your environment configurations manually. The license key is not pre-provisioned on Starter environments.
+
+### Configure New Relic for Starter environments
+
+For Starter environments, you must add the New Relic license key to the environment configuration to enable New Relic reporting. We recommend adding the key to the Staging and Production (master) environments and one other environment of your choice. Only the New Relic license key is required for configuration. You can find information about additional configuration options in the [New Relic reporting] topic in the _Magento User Guide_.
+
+{:.bs-callout-warning}
+Updating the environment configuration triggers a redeployment, which takes your site offline until deployment completes. For Production environments, we recommend completing this work during off-peak hours to avoid service disruptions. See [Working with variables].
+
+{:.prerequisites}
```
Comment (meker12) [inline src/cloud/project/new-relic.md]:
```suggestion
{:.procedure}
Prerequisites
```

File: src/cloud/project/new-relic.md [hunk 2]
```diff
@@ -69,40 +69,53 @@ runtime:
...
 
-{:.procedure}
-To add a license key to an environment:
+   -  Expand the **New Relic Service** section to view the license key.
 
-1. Change to the environment that needs the license key. For Starter accounts, you need to use the license key to set a variable in the environment.
+   -  Copy the license key.
 
-1. Set the variable.
+1. Add the New Relic license key to an environment using an environment variable, or [add it from the Magento Admin]:
```
Comment (hguthrie) [inline src/cloud/project/new-relic.md]:
```suggestion
1. Add the New Relic license key to an environment using the magento-cloud CLI, or [add it from the Magento Admin]:
```

Need to be specific. Saying environment variable may make some people think about the .magento.env.yaml file. or the Project Web UI variables list.

File: src/cloud/project/new-relic.md [hunk 3]
```diff
@@ -245,11 +255,39 @@ The Adobe-generated policies have default notification channels configured to no
 
 Do not modify the Adobe-generated alert policies. We will automatically regenerate and improve them over time, which will overwrite any customizations you make.
 
-If you want to modify the thresholds of an alert condition, create a new alert policy. Then, copy the alert condition to the new policy. See [Update policies or conditions](https://docs.newrelic.com/docs/alerts/new-relic-alerts/configuring-alert-policies/update-or-disable-policies-conditions) in the New Relic documentation.
+If you want to modify the thresholds of an alert condition, create a new alert policy. Then, copy the alert condition to the new policy. See [Update policies or conditions] in the New Relic documentation.
 
 {:.bs-callout-tip}
-See [Alerts concepts and workflow](https://docs.newrelic.com/docs/alerts/new-relic-alerts/getting-started/new-relic-alerts-concepts-workflow) in the New Relic documentation for more detailed information about Alerts, alert policies, and notification channels.
+See [Alerts concepts and workflow] in the New Relic documentation for more detailed information about Alerts, alert policies, and notification channels.
```
Comment (hguthrie) [inline src/cloud/project/new-relic.md]: Might be nice to have a comment here.
```suggestion

<!-- link definitions -->

```

File: src/cloud/project/new-relic.md [hunk 4]
```diff
@@ -8,26 +8,26 @@ functional_areas:
...
+[New Relic for application performance management (APM)] is a software analytics product that helps you analyze and improve application interactions. New Relic APM is available to all {{site.data.var.ece}} accounts and provides the following features:
 
 -  **Focus on specific transactions**—Actively mark and monitor key customer actions in your site, such as adding to the cart, checking out, or processing a payment.
 -  **Database query monitoring**—Locate and monitor database queries affecting performance.
 -  **App Map**—View all application dependencies within your Magento site, extensions, and external services.
--  **Apdex scores**—Evaluate performance and create alerts that identify issues and notify you when they occur, such as site performance affected by a flash sale or web event. [Apdex score](https://docs.newrelic.com/docs/apm/new-relic-apm/apdex/apdex-measure-user-satisfaction).
+-  **Apdex scores**—Evaluate performance and create alerts that identify issues and notify you when they occur, such as site performance affected by a flash sale or web event. [Apdex score].
```
Comment (hguthrie) [inline src/cloud/project/new-relic.md]:
```suggestion
-  **Apdex scores**—Evaluate performance and create alerts that identify issues and notify you when they occur, such as site performance affected by a flash sale or web event. See [Apdex score].
```

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**